components/public-api/gitpod/v1/organization.proto

```diff
@@ -18,9 +18,9 @@ message OrganizationMember {
   string user_id = 1;
   OrganizationRole role = 2;
   google.protobuf.Timestamp member_since = 3;
-  optional string avatar_url = 4;
-  optional string full_name = 5;
-  optional string email = 6;
+  string avatar_url = 4;
```

Comment:
in this file I drop optional keywords from everywhere but to make update calls partial